youtube_dl/postprocessor/embedthumbnail.py

```diff
@@ -3,17 +3,18 @@
 
 
 import os
-import subprocess
-
 from .ffmpeg import FFmpegPostProcessor
 
+try:
+    import imghdr
+    from mutagen.mp4 import MP4, MP4Cover, MP4MetadataError
+except ImportError:
+    print("[embedthumbnail] MP4 thumbnail embedding cannot be done, mutagen is missing.")
```

Comment:
No. It must be a soft dependency. Message requesting installation of `AtomicParsley` or `mutagen` should only be output when thumbnail embedding is requested and neither of these dependencies is found.